README.md

```diff
@@ -323,6 +342,37 @@ if (validateHuman(data.honeypot)) {  //if form is filled, form will not be submi
 ```
 
 
+## Frequently Asked Questions (FAQ's)
+
+1. _How can I get help using this tutorial?_
+
+- This tutorial is meant to help people learn how to use web technologies and make simple, customizable forms with the Google Script API. If you have been unable to get a working example of your code, please check each of the tutorial steps first. You may have missed or glossed over a detail, and this can save both you, and us, time, when trying to help you. Furthermore, if you do this and are still unable to fix your problem, feel free to submit a GitHub issue, but try to include an example of the problem. Highlight what steps you have gone through, what exactly isn't working and why it isn't working. What do you expect to happen? Having an example of the problem, say a GitHub pages or a CodePen, goes a long way since it lowers the barrier for someone to take your code and see what the issue is. If you want to get a faster response, I would highly encourage putting together an example.
```

Comment:
Minor minor things here:
+ This is a massive paragraph which is harder to edit/comment on - you might want to consider semantic line breaks (or breaking at the 80 character line mark) which won't affect the visual side of this but will make the markdown easier to follow
+ We've found it's often fruitful to link 'submit a github issue' to https://github.com/dwyl/html-form-send-email-via-google-script-without-server/issues/new for ease of access, though I'm not 'ease of access' is what you're hoping for here! 😆

Reply:
Cool, I can definitely take a quick editing pass on this for sure. Not sure how to best have line breaks on a bulleted list item in Markdown offhand though. I can fix this up and assign when ready!